common/util/init_command_line.cc

```diff
@@ -19,11 +19,26 @@
 #include "absl/flags/flag.h"
 #include "absl/flags/parse.h"
 #include "absl/flags/usage.h"
+#include "absl/flags/usage_config.h"
+#include "common/util/verible_build_version.h"
 
 namespace verible {
 
+std::string GetBuildVersion() {
```

Comment:
`constexpr absl::string_view`? (all compile-time constant string literals)

Reply:
No, this is the rquired std::function signature.